server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/LocalStorageManager.scala

```diff
@@ -209,8 +209,10 @@ private[worker] final class Flusher(
 }
 
 private[worker] final class LocalStorageManager(
-  conf: RssConf,
-  workerSource: AbstractSource) extends DeviceObserver with Logging with MemoryTrackerListener{
+    conf: RssConf,
+    workerSource: AbstractSource,
+    partitionLocationInfo: PartitionLocationInfo)
```

Comment:
There is no need for partitionLocationInfo, I think there is an alternative solution to iterate all file writer instances.

Reply:
Removed